tools/pylib/_boutcore_build/boutcore.pyx.in

```diff
@@ -623,6 +623,83 @@ cdef class Laplacian:
         """
         return f3dFromObj(self.cobj.solve(a.cobj[0],b.cobj[0]))
 
+    def setCoefA(self,Field3D A):
+        """
+        Set the 'A' coefficient of the Laplacian solver
+
+        Parameters
+        ----------
+        A : Field3D
+            Field to set as coefficient
```

Comment:
Comment not very descriptive. Can the equation be included, to show which coefficient each of these corresponds to?